deploy/gk-deploy

```diff
@@ -75,6 +77,16 @@ Options:
               device or node, then restart with --load to skip gluster
               deployment and re-run topology load command.
 
+  --admin-key ADMIN_KEY
+              Secret for heketi administrator user. Heketi admin has access to all 
+	      the APIs. admin key: string, Shared secret. 
+	      Eg: MySecret
```

Comment:
There are tab characters in the whitespace on these two lines, so for me the indentation looks messed up. Replace them with spaces so we get consistent indentation. :)